pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -1362,35 +1364,111 @@ struct ScatterOpConversion : public OpConversionPattern<tile::ScatterOp> {
 
     // Load the location value from the indices tensor.
     // Create an affine map for loading the index, using leading counters.
-    size_t idxDims = indices.getType().cast<MemRefType>().getShape().size();
+    int axis = *(op.axis().getRawData());
```

Comment:
The convention is defining axis as an IndexAttr, which is signless. It might be better to  deal negative axis earlier. In gather, currently negative axis is converted  as soon as passing to the gather class.

Reply:
Got it. Will deal with negative number in the way as gather does.